### ISSUE TITLE
Game version mismatch?

### ISSUE THREAD

**Issue by M4YON** [HarukaMa/palworld-xgp-import#1]
The save successfully converts to a WGS file and container and is readable by the game. Upon trying to load the save file the world begins to load I even gained an achievement the first time I tried loading in but then it immediately crashes. Possible version mismatch issue due to game pass being on v0.1.1.2 and steam on v0.1.3.0?

**Comment by HarukaMa** [comment]
Great point, that's definitely possible. I don't own steam version so I only tried with a save downloaded from internet, and that apparently worked. 

If that's the case, then the best I can do is to try to parse the save file and notify the user. I'm not sure what's the save file's format though.

**Comment by M4YON** [comment]
Okay after some messing around I tested a downloaded a save from nexus and it worked great. Then i changed the settings on the downloaded save to multiplayer and reloaded the save instantly crashed. So I went back to my save file from v0.1.3.0 turned off multiplayer so it was offline and it worked and loaded. I'm not sure why turning multiplayer on is crashing things but for now I'm happy to be solo offline.
